bin/tap-difflet

```diff
@@ -19,8 +19,67 @@ var tap = parser();
 var out = through();
 var dup = duplexer(tap, out);
 
+
+/* Pessimism */
+
+function userSpecifiedPessimistic (args) {
+  for (var ai=2; ai < args.length; ++ai) {
+    if (args[ai] === '-p' || args[ai] === '--pessimist') {
+      return true;
+    }
+  }  
+
+  return false;
+};
+
+
+if (userSpecifiedPessimistic(process.argv)) {
+  var anAssertFailed = true;
+  var originalOut = out;
+  var buffer = null;
+  tap.on('comment', function(comment) {
+    if (buffer && anAssertFailed) {
+      for (var bi=0; bi<buffer.length; ++bi) {
+        originalOut.push(buffer[bi]);
+      }
+    }
+    
+    buffer = null;
+    anAssertFailed = false;
+    out = originalOut;
+    
+    if (commentType(comment) === '') {
+      buffer = [];
+      out = buffer;
+    }    
+  });
+  
+  tap.on('assert', function(res) {
+    if (!res.ok) {
+      anAssertFailed = true;  
+    }  
+  });
+}
+
 /* Helpers */
 
+function commentType(comment) {
+  if (/^tests\s+[0-9]+$/gi.test(comment)) {
+    return 'tests';
+  }
+  else if (/^pass\s+[0-9]+$/gi.test(comment)) {
+      return 'pass';
```

Comment:
you have funky indentation, 2 spaces above, 4 here. This library has chosen 2 (but clearly missed a few places) can you stick with 2 (and possibly cleanup other places that were missed over time)